packages/@okta/vuepress-site/guides/sign-users-out/sign-out-of-your-app/aspnet/localsignout.md

```diff
@@ -0,0 +1,23 @@
+Open the controller where you will handle the sign out process and add a `SignOut` method to delete the application cookies:
+
+```csharp
+public class AccountController : Controller
+{
+    [HttpPost]
```

Comment:
Can you add `[ValidateAntiForgeryToken]` here (and make sure it still works)?

We should encourage the best practice of CSRF protection on any POST route.

Reply:
I suggest to add this once we fix : [okta-aspnet: Fix NameClaimType to solve ValidateAntiForgeryToken issue on ASP.NET MVC](https://oktainc.atlassian.net/browse/OKTA-212374).